### PR TITLE
docs: (IAC-916) V4M monitoring and logging fails to install with open-source K8s because of the missing v4m SC

### PR DESCRIPTION
**Pull request by dhoucgitter**
# Changes
Adds required instructions for setting `V4M_STORAGECLASS` value to `local-storage` in ansible-vars.yaml when using [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git) to deploy `cluster-logging` and `cluster-monitoring` applications.

### CODE REVIEW

**Review by thpang — CHANGES_REQUESTED** [review]

File: docs/REQUIREMENTS.md [hunk 1]
```diff
@@ -666,6 +666,13 @@ INGRESS_NGINX_CONFIG:
 NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas
 ```
 
+You must set the value for `V4M_STORAGECLASS` to `local-storage` in your ansible-vars.yaml file as shown below prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).
+
```
[the remark this answers is not in the document]
Comment (dhoucgitter) [reply docs/REQUIREMENTS.md]: Applied updates from Jira review comments. Ready for re-review.

**Review by thpang — APPROVED** [review]

**Review by jarpat — DISMISSED** [review]

**Review by riragh — APPROVED** [review]

File: docs/REQUIREMENTS.md [hunk 2]
```diff
@@ -666,6 +666,14 @@ INGRESS_NGINX_CONFIG:
 NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas
 ```
 
+You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown below to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).
+
+```yaml
+## Logging and Monitoring
+V4M_STORAGECLASS = local-storage
+```
```
Comment (thpang) [inline docs/REQUIREMENTS.md:674]: I would prefer to have all of the code in an easy snippet that folks could copy and paste into their ansible vars file for the deployment tooling. If needed we should provide guidance to these items after the snippet.

For example the snippet should look like:

```yaml
## 3rd Party
### Ingress Controller
INGRESS_NGINX_CONFIG:
  controller:
    service:
      externalTrafficPolicy: Cluster or Local # Refer to Load Balancer type table above
      # loadBalancerIP: <your static ip> # Assigns a specific IP for your loadBalancer
      loadBalancerSourceRanges: [] # Not supported on open source kubernetes - https://kubernetes.io/docs/reference/kubernetes-api/service-resources/service-v1/
      annotations:

### NFS Subdir External Provisioner - SAS default storage class
# Updates to support open source Kubernetes 
NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas

## Logging and Monitoring
V4M_STORAGECLASS = local-storage
```
**NOTE**: You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown below to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).

This is much cleaner and easier for folks to use and digest.

File: docs/REQUIREMENTS.md [hunk 3]
```diff
@@ -664,8 +664,15 @@ INGRESS_NGINX_CONFIG:
...
 NFS_CLIENT_NAME: nfs-subdir-external-provisioner-sas
+
+## Logging and Monitoring
+V4M_STORAGECLASS = local-storage
 ```
 
+You must explicitly set the value for `V4M_STORAGECLASS` in your ansible-vars.yaml file as shown above to a pre-existing Storage Class (for example: `local-storage`) prior to installing `cluster-logging` or `cluster-monitoring` with [viya4-deployment](https://github.com/sassoftware/viya4-deployment.git).
+
+While other storage classes can be used, the `local-storage` class is **recommended** for the Viya Monitoring and Logging tools.
+
```
Comment (thpang) [inline docs/REQUIREMENTS.md:675]: I myself would still call this out as a NOTE to draw attention to the item. But it's fine as is it right now. I'll approve.

**Review by thpang — APPROVED** [review]
Made a comment on the change, but approving.

**Review by riragh — APPROVED** [review]

**Pull request merged**